ui/src/pages/backoffice/Loan/LoanSearch/LoanDates.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { invenioConfig } from '@config';
+import { Icon, List } from 'semantic-ui-react';
+
+export class LoanDates extends Component {
```

Comment:
@topless for new components, may I ask that we always include all the PropTypes and defaults?
I would also pass, as we discussed, the minim amount of info necessary, maybe in this case only the `metadata` object.
I found that it was really helpful while refactoring. I will pay attention to this also in my code (need to re-check my PRs)

Reply:
@ntarocco we can start filling them up, those who miss. On top of that instead of using the generic and without any value object, maybe we could use something like this https://github.com/dowjones/react-json-schema-proptypes so we start working with proper propTypes.shapes and "stronger" type safety.

Reply:
interesting idea, but it will require some changes in our base code. I think that what we are missing is just more strict prop types (e.g. if you pass something not defined in the prop types, it should complain), which is discussed but not yet addressed [here](libraryMetadata).